.gitlab-ci.yml

```diff
@@ -35,72 +39,117 @@ build-simnetscripts-image-and-push-temporarly-in-paritypr:
         quay.io/buildah/stable \
             buildah bud  \
               --layers=true  \
-              --cache-from  "$CONTAINER_IMAGE:$CI_COMMIT_SHORT_SHA" \
-              --tag         "$CONTAINER_IMAGE:$CI_COMMIT_SHORT_SHA" \
+              --cache-from  "${CONTAINER_IMAGE}:${CONTAINER_IMAGE_TAG}" \
+              --tag         "${CONTAINER_IMAGE}:${CONTAINER_IMAGE_TAG}" \
               /build 
-      echo "LOG INFO step 5 image $CONTAINER_IMAGE:$CI_COMMIT_SHORT_SHA built successfully"
+      echo "LOG INFO image ${CONTAINER_IMAGE}:${CONTAINER_IMAGE_TAG} built successfully"
       echo "$DOCKER_SIMNET_PASS" \
            | podman  login --username "$DOCKER_SIMNET_USER" --password-stdin docker.io
-      podman push --format=v2s2 "$CONTAINER_IMAGE:$CI_COMMIT_SHORT_SHA"
-      echo "LOG INFO step 6 image $CONTAINER_IMAGE:$CI_COMMIT_SHORT_SHA pushed successfully"
+      podman push --format=v2s2 "$CONTAINER_IMAGE:$CONTAINER_IMAGE_TAG"
+      echo "LOG INFO ${CONTAINER_IMAGE}:${CONTAINER_IMAGE_TAG} pushed successfully"
       echo "LOG check if need to cleanup dangling images"
       df -h | grep /var/lib/containers
       CACHE_USE_PERCENT="$(df -h | grep -vE '/var/lib/containers/.*' | grep /var/lib/containers | awk '{print $5}' | sed 's/%//g')"
-      test -n "$CACHE_USE_PERCENT" \
+      test -n "${CACHE_USE_PERCENT}" \
               || ( echo "This variable can't be emtpy" ; exit 1 )
-      if test "$CACHE_USE_PERCENT" -ge 80 ; then 
-        echo "LOG INFO need to cleanup cache, value: $CACHE_USE_PERCENT"
+      if test "${CACHE_USE_PERCENT}" -ge 80 ; then 
+        echo "LOG INFO need to cleanup cache, value: ${CACHE_USE_PERCENT}"
         time podman rmi -f $(podman images -q -f "dangling=true")
       else 
-        echo "LOG INFO No Cleanup needed this time value: $CACHE_USE_PERCENT"
+        echo "LOG INFO No Cleanup needed this time value: ${CACHE_USE_PERCENT}"
       fi
   retry: 1
   after_script:
     - podman logout docker.io
 
+test-current-latest-container-image:
+  stage:                           build
+  tags:
+    - parity-simnetscripts
+  variables:
+    IMAGE_UNDER_TEST:              "paritytech/simnetscripts"
+    IMAGE_UNDER_TEST_TAG:          "latest"
+    PARACHAINS_IMAGE:              "docker.io/paritypr/synth-wave"
+    # there is no master tag for docker.io/paritypr/synth-wave
+    # Error response from daemon: manifest for paritypr/synth-wave:master not found: manifest unknown: manifest unknown
+    PARACHAINS_IMAGE_TAG:          "2714"
```

Comment:
it's merged already

Reply:
`radu@radu-x1-carbon:~/p/simnet_scripts$ docker pull docker.io/paritypr/synth-wave:master
Error response from daemon: manifest for paritypr/synth-wave:master not found: manifest unknown: manifest unknown
`